### ISSUE TITLE
KeyError: 'assembly_accession'

### ISSUE THREAD

**Issue by tseemann** [kblin/ncbi-genome-download#2]
```
% ncbi-genome-download -V
0.1.0

% ncbi-genome-download -v viral

INFO: Starting new HTTP connection (1): ftp.ncbi.nih.gov
Traceback (most recent call last):
  File "/bio/linuxbrew/bin/ncbi-genome-download", line 11, in <module>
    sys.exit(main())
  File "/bio/linuxbrew/Cellar/python/2.7.11/lib/python2.7/site-packages/ncbi_genome_download/__main__.py", line 43, in main
    ncbi_genome_download.download(args)
  File "/bio/linuxbrew/Cellar/python/2.7.11/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 21, in download
    _download(args.section, args.domain, args.uri, args.output)
  File "/bio/linuxbrew/Cellar/python/2.7.11/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 29, in _download
    download_entry(entry, section, domain, uri, output)
  File "/bio/linuxbrew/Cellar/python/2.7.11/lib/python2.7/site-packages/ncbi_genome_download/core.py", line 56, in download_entry
    logging.info('Downloading record %r', entry['assembly_accession'])
KeyError: 'assembly_accession'
```

**Comment by kblin** [comment]
Hi @tseemann, I can confirm this. Somebody at NCBI apparently thought it was funny to add an extra line to this file, breaking the CSV reader autodetection of the column names. Working on a fix.

**Comment by kblin** [comment]
That is, the start of ftp://ftp.ncbi.nih.gov/genomes/refseq/viral/assembly_summary.txt is:
```
#   See ftp://ftp.ncbi.nlm.nih.gov/genomes/README_assembly_summary.txt for a description of the columns in this file.
# assembly_accession	bioproject	biosample	wgs_master	refseq_category	taxid	species_taxid	organism_name	infraspecific_name	isolate	version_status	assembly_level	release_type	genome_rep	seq_rel_date	asm_name	submitter	gbrs_paired_asm	paired_asm_comp	ftp_path	excluded_from_refseq
GCF_000839185.1	PRJNA14174			na	10243	10243	Cowpox virus	strain=Brighton Red		latest	Complete Genome	Major	Full	2003/05/16	ViralProj14174		na	na	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000839185.1_ViralProj14174	
GCF_000857045.1	PRJNA15142			na	619591	10244	Monkeypox virus Zaire-96-I-16	strain=Zaire-96-I-16		latest	Complete Genome	Major	Full	2001/12/12	ViralProj15142		na	na	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000857045.1_ViralProj15142	
GCF_000860085.1	PRJNA15241			na	10245	10245	Vaccinia virus	strain=WR (Western Reserve)		latest	Complete Genome	Major	Full	2003/04/10	ViralProj15241		na	na	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000860085.1_ViralProj15241	
GCF_000859885.1	PRJNA15197			na	10255	10255	Variola virus	strain=India-1967, ssp. major	Ind3	latest	Complete Genome	Major	Full	1993/06/16	ViralProj15197		na	na	ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000859885.1_ViralProj15197
```

**Comment by kblin** [comment]
Closed by d1e8bfe
